doc/nrf/known_issues.rst

```diff
@@ -185,6 +185,14 @@ nRF5340
 
 .. rst-class:: v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0
 
+NCSDK-9786: Wrong FLASH_PAGE_ERASE_MAX_TIME_US for nrf53 net core
+  FLASH_PAGE_ERASE_MAX_TIME_US is the execution window duration when doing the flash operation synchronously along the radio operations
+  (SOC_FLASH_NRF_PARTIAL_ERASE not enabled); the FLASH_PAGE_ERASE_MAX_TIME_US value of cpunet is lower than required.
+  For this reason, if CONFIG_SOC_FLASH_NRF_RADIO_SYNC_MPSL=y and CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE=n, doing flash_erase op on nrf5340 net
+  core will result in mpsl timeslot OVERSTAYED assert.
```

Comment:
```suggestion
NCSDK-9786: Wrong FLASH_PAGE_ERASE_MAX_TIME_US for the nRF53 network core
  ``FLASH_PAGE_ERASE_MAX_TIME_US`` defines the execution window duration when doing the flash operation synchronously along the radio operations (:option:`CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE` not enabled).
  
  The ``FLASH_PAGE_ERASE_MAX_TIME_US`` value of the nRF53 network core is lower than required.
  For this reason, if :option:`CONFIG_SOC_FLASH_NRF_RADIO_SYNC_MPSL` is set to ``y`` and :option:`CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE` is set to ``n``, a flash erase operation on the nRF5340 network core will result in an MPSL timeslot OVERSTAYED assert.
```

Reply:
Is there a workaround? Maybe fixing the FLASH_PAGE_ERASE_MAX_TIME_US value (or should users not touch that)?

Reply:
Another PR for fixing the FLASH_PAGE_ERASE_MAX_TIME_US on zephyr side has been merged today.

Reply:
That won't help customers though that use one of the affected releases - the idea is to tell them what they can do to work around the issue.